docs/src/behavior.md

```diff
@@ -26,8 +28,9 @@ docsearch({
 
 This method is called when a suggestion is selected (either from a click or a
 keystroke). By default, DocSearch will display links redirecting to the results
-page, at the related anchor. You can override it to add your own behavior.
-Please note that you can already open a new tab thanks to the `CMD/CTRL + Click`
+page, at the related position (using anchor). You can override results (hit) to
```

Comment:
I don't get the `you can override results (hit)` part. First I think we need to be consistent and either use records or hit, but not both. Second, it's not the actual records you can overwrite to change the behavior, but what is happening when you click on them (ie, it's a function you change, not a data object).

Reply:
That's right, I think using both term here is nice to introduce Algolia global documentation